qubes-kde-dom0/qubes-kde-dom0.spec

```diff
@@ -2,13 +2,16 @@
 
 %if 0%{?qubes_builder}
 %define _sourcedir %(pwd)/qubes-kde-dom0
+%define qubes_kde_dom0_version %(cat version)
+%define breeze_version %(cat version_breeze)
+%define breeze_epoch %(cat epoch_breeze)
```

Comment:
What about putting `5.5.4` in `version` file? It is no longer KDE 4.12 and the only other package using this is `kde-baseapps`, which now seems to be versioned independently of KDE, so may be switched to verbose version in spec file itself.
Or, if there is no longer such thing as synchronized KDE packages version, better remove `version` file completely and place appropriate version in spec files.